leakage-test/leakage-test_shacl.ttl

```diff
@@ -0,0 +1,145 @@
+        sh:maxCount 1 ;
+        sh:node leakage-test:MaximumPressureShape;
+        sh:path leakage-test:MaximumPressure ] ,
+    [ sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:node leakage-test:MinimumPressureShape;
+        sh:path leakage-test:MinimumPressure ]; 
+    sh:targetClass leakage-test:LeakageTest .
+
+leakage-test:MachineParameterShape
+    a sh:NodeShape ;
+    sh:property [ sh:minCount 1 ;
+	sh:maxCount 1 ] ;
+    sh:targetClass leakage-test:MachineParameter .
+
+    leakage-test:ProcessParameterShape
+    a sh:NodeShape ;
+    sh:property [ sh:minCount 1 ;
+        sh:maxCount 1 ] ;
+    sh:targetClass leakage-test:ProcessParameter .
+
+leakage-test:DownholderShape
+    a sh:NodeShape ;
+    sh:property [ skos:example "1" ;
+        sh:datatype xsd:integer ;
+	sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:message "Validation of setpoint failed!"@en ;
+        sh:name "setpoint"@en ;
+        sh:path leakage-test:setpoint ] ;
+    sh:targetClass leakage-test:Downholder .
+
+leakage-test:CodingTestAdapterShape
+    a sh:NodeShape ;
+    sh:property [ skos:example "1" ;
+        sh:datatype xsd:integer ;
+	sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:message "Validation of setpoint failed!"@en ;
+        sh:name "setpoint"@en ;
+        sh:path leakage-test:setpoint ] ; 
+    sh:targetClass leakage-test:CodingTestAdapter .
+
+leakage-test:CodingRFIDShape
+    a sh:NodeShape ;
+    sh:property [ skos:example "16" ;
+        sh:datatype xsd:integer ;
+	sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:message "Validation of setpoint failed!"@en ;
+        sh:name "setpoint"@en ;
+        sh:path leakage-test:setpoint ] ; 
+    sh:targetClass leakage-test:CodingRFID .
+
+leakage-test:MaximumPressureShape
+    a sh:NodeShape ;
+    sh:property [ skos:example "bar" ;
+        sh:datatype xsd:string ;
+	sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:message "Validation of unit failed!"@en ;
+        sh:name "unit"@en ;
+        sh:path leakage-test:unit ] ,
+    [ skos:example "0" ;
+        sh:datatype xsd:decimal ;
+	sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:message "Validation of upperTolerance failed!"@en ;
+        sh:name "upperTolerance"@en ;
+        sh:path leakage-test:upperTolerance ] ,
+     [ skos:example "2.7";
+        sh:datatype xsd:decimal ;
+	sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:message "Validation of setpoint failed!"@en ;
+        sh:name "setpoint"@en ;
+        sh:path leakage-test:setpoint ] ,
+    [ skos:example "0" ;
+        sh:datatype xsd:decimal ;
+	sh:minCount 1 ;
+        sh:maxCount 1 ;
+        sh:message "Validation of lowerTolerance failed!"@en ;
+        sh:name "lowerTolerance"@en ;
+        sh:path leakage-test:lowerTolerance ] ; 
+    sh:targetClass leakage-test:MaximumPressure .
+    
+leakage-test:MinimumPressureShape
+    a sh:NodeShape ;
+    sh:property [ skos:example "bar" ;
+        sh:datatype xsd:string ;
+	    sh:minCount 1 ;
```

Comment:
can you fix the indentation here to match the others? Below there are some more, each time at sh:minCount

Reply:
there are still indentation problems in the file.
the reason for that is that you have both tabs and spaces as indentation.
Please switch over to spaces then it is similar to the other shacl shapes.
Example:
<img width="359" alt="image" src="https://github.com/user-attachments/assets/8e733be1-be6c-4ad3-8d14-93213d04cfc8">